src/Laser.h

```diff
@@ -1,5 +1,9 @@
-/*
-  Laser.h - Simple class for laser module
+/**
+  Calico Home Security System, Laser.h
+  Purpose: Manges the lASER and creats a LASER object
```

Comment:
I'll get right on fixing these issues, I guess I was miss understanding the command function a little bit, shouldn't be to hard to update these real quick.